webserver/views/user.py

```diff
@@ -98,7 +104,35 @@ def import_data():
                            lastfm_username=lastfm_username)
 
 
+@user_bp.route("/export")
+@login_required
+def export_data():
+    """ Exporting the data to various formats, like, json """
+    filetype = request.args.get("type")
+    if filetype == "json":
```

Comment:
As of now i don't think there are any concrete plans to add new export format. Also, actually we just need a single format to export.
Adding POST request.